oscarapi/tests/testbasket.py

```diff
@@ -0,0 +1,939 @@
+import json
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from oscar.core.loading import get_model
+
+from oscarapi.tests.utils import APITest
+
+
+Basket = get_model('basket', 'Basket')
+
+
+class BasketTest(APITest):
+    fixtures = [
+        'product', 'productcategory', 'productattribute', 'productclass',
+        'productattributevalue', 'category', 'attributeoptiongroup', 'attributeoption',
+        'stockrecord', 'partner'
+    ]
+
```

Comment:
Never heard of it, but what problem we are having with the fixtures right now, would that solve?